packages/plugin-analytic-google/src/GoogleAnalytics4.js

```diff
@@ -137,16 +137,16 @@ export default class GoogleAnalytics4 extends AbstractAnalytic {
    */
   _getPageViewData(pageData) {
     const page_location = this._window.getUrl();
```

Comment:
Nechceme ty optional chaining operatory i tady a dole v te metode `_createGlobalDefinition()`?

Reply:
Tady jsem dala, ale v `_createGlobalDefinition()` to pak vůbec nedává smysl. Mohla bych tam dát prázdný return, ale zdá se mi lepší, když to přímo spadne.